src/CUBLAS.jl

```diff
@@ -11,8 +11,11 @@ module CUBLAS
 
 importall Base.LinAlg.BLAS
 
-using CUDArt
-using CUDArt.rt.cudaStream_t
+using CUDAdrv: OwnedPtr, CuArray, CuVector, CuMatrix
```

Comment:
`OwnedPtr` is an implementation detail that shouldn't be used downstream. for `cudacall` you can just use `Ptr`. why do you need it?